guides/common/modules/con_content-views-in-projectname.adoc

```diff
@@ -0,0 +1,46 @@
+[id="Content-Views-in-{ProjectNameID}_{context}"]
+= Content Views in {ProjectName}
+
+A _content view_ defines a deliberately curated subset of content that your hosts can access.
+Each content view creates a set of repositories across each environment.
+The {ProjectServer} stores and manages these repositories.
+You can use content views to define which software versions a particular environment or {SmartProxyServer} uses.
```

Comment:
Here I would suggest using a passive to improve readability. This must be a first for me.

```suggestion

```

Or maybe better

`You can define the software versions used by a particular environment or {SmartProxyServer} by using content views.`